src/Confluent.SchemaRegistry.Rules/BuiltinLibrary.cs

```diff
@@ -7,15 +7,15 @@ public class BuiltinLibrary : ILibrary
 
         public virtual IList<EnvOption> CompileOptions
         {
-            get => new List<EnvOption> { IEnvOption.Declarations(BuiltinDeclarations.Create()) };
+            get => new List<EnvOption> { EnvOptions.Declarations(BuiltinDeclarations.Create()) };
         }
 
         public virtual IList<ProgramOption> ProgramOptions
         {
             get => new List<ProgramOption>
             {
-                IProgramOption.EvalOptions(EvalOption.OptOptimize),
-                IProgramOption.Functions(BuiltinOverload.Create())
+                Cel.ProgramOptions.EvalOptions(EvalOption.OptOptimize),
```

Comment:
What is the reason behind this change? Is IProgramOption deprecated in favor of cel.ProgramOptions

Reply:
Yes